services/cosi-driver-nutanix/0.0.4/cosi-driver-nutanix.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: HelmRepository
+metadata:
+  name: cosi-driver-nutanix
+  namespace: kommander-flux
+spec:
+  interval: 5m0s
+  # TODO(takirala): Use HelmRepository instead of OCIRepository so that this works in air-gapped environments.
+  url: oci://ghcr.io/nutanix-cloud-native/chart/cosi-driver-nutanix
+  type: oci
```

Comment:
Why does this live in here rather than in `helm-repositories` directory?

Reply:
Fixed in 1553830acd4b322045b346dbe06f42182c9e6bff